forge-game/src/main/java/forge/game/ability/effects/CharmEffect.java

```diff
@@ -271,10 +271,14 @@ public static void chainAbilities(SpellAbility sa, List<AbilitySub> chosen) {
         // Sort Chosen by SA order
         chosen.sort(Comparator.comparingInt(o -> o.getSVarInt("CharmOrder")));
 
+        int indx = 1;
         for (AbilitySub sub : chosen) {
             // Clone the chosen, just in case the same subAb gets chosen multiple times
             AbilitySub clone = (AbilitySub)sub.copy(sa.getActivatingPlayer());
 
+            clone.setSVar("CharmOrder", Integer.toString(indx));
```

Comment:
Shouldn't CharmOrder be copied too? 🤔

Reply:
it sure is
that's why if you only chose the second mode your new logic would calculcate CharmOrder 2 - 1 and raise the cost